src/user-research/getting-informed-consent-for-user-research.md

```diff
@@ -86,53 +86,53 @@ In fully confidential research, our client does not learn the identity of our pa
+For user research sessions where we meet the participant in person, we can also:
 
 * give the participant a printed copy of the consent form to sign
 * share a consent form in an accessible digital format the participant can read and respond to using their assistive technology
 * read the consent form aloud and collect their consent on an audio or video recording
 
-Whichever approach you are taking, make sure the participants have time to read the research and consent information carefully, and that you give them an explicit opportunity to ask questions.
+Whichever approach we are taking, we make sure the participants have time to read the research and consent information carefully. And we give them an explicit opportunity to ask questions.
 
 ### During the session
 
-At the start of the session:
+At the start of the session we:
 
 * remind the participant that their participation is voluntary and they can stop or take a break at any time
-* remind them about the different ways you are recording the session, and about anyone observing or listening in
+* remind them about the different ways we are recording the session, and about anyone observing or listening in
 * ask them to confirm their consent for the recording
 
-As the session continues, actively make sure you have the participant’s consent. For example, check in with the participant if they look concerned about an activity, or if you’re on a visit and want to take a photo in a new area.
+As the session continues, we actively make sure we have the participant’s consent. For example, by checking in with the participant if they look concerned about an activity, or if asking for permission when we’re on a visit and want to take a photo in a new area.
 
-As a researcher, you can also pause to take a break, or finish a session if you are uncomfortable.
+As a researcher, we can also pause to take a break, or finish a session if we are uncomfortable.
 
 ### Keeping a record of informed consent
 
-For email and similar responses, keep a copy of the research information and consent sheet you used, along with a copy of each participant's reply, in the folder with the research data you collect.
+For email and similar responses, we keep a copy of the research information and consent sheet we used, along with a copy of each participant's reply, in the folder with the research data we collect.
 
-For paper consent forms, scan or photograph the signed consent form and keep the image file. Avoid [using your own phone or other personal device](https://playbook.dxw.com/user-research/taking-notes-and-recording-research-sessions/#using-a-dxw-provided-handheld-device) for this.
+For paper consent forms, we scan or photograph the signed consent form and keep the image file. We avoid [using our own phone or other personal device](https://playbook.dxw.com/user-research/taking-notes-and-recording-research-sessions/#using-a-dxw-provided-handheld-device) for this.
```

Comment:
And what happens to the signed paper consent forms? where are we storing them? taking into account that we are a 'work from home' first agency.

Reply:
I've added a note on disposing of the paper.